application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -1809,7 +1809,7 @@ public function add_participants($sSessionKey, $iSurveyID, $aParticipantData, $b
                 if ($bCreateToken) {
                     $token->generateToken();
                 }
-                if ($token->encryptSave()) {
+                if ($token->encryptSave(true)) {
```

Comment:
We need a unit-test for this, I think. It's not clear when and how validation can be used with encryption.

Reply:
I agree unit-test could be a good way approach.

Reply:
Can you make one, in this PR, please? Ping me if you need help or instructions. There already is a test for the remote API that you can look at.

Reply:
What you added looks good, but I was thinking of a new test in this file: https://github.com/LimeSurvey/LimeSurvey/blob/master/tests/unit/helpers/RemoteControlTest.php

For add_participants RPC call. Sorry for being vague. But, more test code is always good! :)

Reply:
Added

Reply:
Great, will try it!